apps/funke/app.config.js

```diff
@@ -6,23 +6,14 @@ const variants = {
   development: {
     bundle: '.dev',
     name: ' (Dev)',
-    trustedCertificates: [
```

Comment:
Ill likely encounter it later, but why was this removed?

Reply:
I just couldn't get it to work. The value was not availalbe in the app. So after spending an hour on it, I just moved it to constants file